app/controllers/decidim/extended_socio_demographic_authorization_handler/autocomplete_controller.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module Decidim
+  module ExtendedSocioDemographicAuthorizationHandler
+    class AutocompleteController < ApplicationController
+      def city
+        render json: JSON.dump(Decidim::ExtendedSocioDemographicAuthorizationHandler::AutocompleteService.for(params[:postal_code]))
```

Comment:
```suggestion
        render json: Decidim::ExtendedSocioDemographicAuthorizationHandler::AutocompleteService.for(params[:postal_code])
```

`JSON.dump` is not necessary here